src/map/entities/charentity.cpp

```diff
@@ -234,6 +234,8 @@ CCharEntity::CCharEntity()
 
     m_Substate = CHAR_SUBSTATE::SUBSTATE_NONE;
 
+    m_TimeLastMemberJoined = server_clock::now() - std::chrono::seconds(120); // default time
```

Comment:
Now that this lives in char entity, this could have a better name

Reply:
Had a small discussion over this, the new var name seems to fit better.